app/controllers/solidus_auth_devise/controller_helpers/deprecated_routes.rb

```diff
@@ -0,0 +1,36 @@
+# frozen_string_literal: true
```

Comment:
I'm not convinced by where this file is placed. Why it's in `app/controllers`? This kind of helper is more something for `libs` IMO, similar to what we do in core.

Reply:
I put it there because it plays nice with autoload, maybe we could put it in `app/controllers/concerns/solidus_auth_devise/deprecated_routes.rb` instead?

Reply:
Yep, better!